static/app/api.tsx

```diff
@@ -275,6 +279,7 @@ export class Client {
     this.baseUrl = options.baseUrl ?? '/api/0';
     this.headers = options.headers ?? Client.JSON_HEADERS;
     this.activeRequests = {};
+    this.credentials = options.credentials ?? 'include';
```

Comment:
this was causing a `cross-origin` issue when fetching the sentry release registry